### PR TITLE
Update style to v9.2.3

### PR DESCRIPTION
**Pull request by rgaiacs**
A few changes on style.

### CODE REVIEW

**Comment by gdevenyi** [comment]
There's some library carpentry stuff as well as some commented out code from other lessons riding along, should that be there?

File: CONDUCT.md [hunk 1]
```diff
@@ -1,44 +0,0 @@
----
```
Comment (rgaiacs) [inline CONDUCT.md]: This need to be undone.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ and to meet some of our community members.
 ## Where to Contribute
 
 1.  If you wish to change this lesson,
-    please work in <https://github.com/swcarpentry/shell-novice>,
-    which can be viewed at <https://swcarpentry.github.io/shell-novice>.
+    please work in <https://github.com/swcarpentry/FIXME>,
```
Comment (rgaiacs) [inline CONTRIBUTING.md]: This need to be undone.

File: _includes/dc/intro.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<p>
```
Comment (rgaiacs) [inline _includes/dc/intro.html]: This is OK. The styles include things for the workshop template.

File: _includes/dc/schedule.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/dc/schedule.html]: This is OK. The styles include things for the workshop template.

File: _includes/dc/syllabus.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/dc/syllabus.html]: This is OK. The styles include things for the workshop template.

File: _includes/dc/who.html [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<p id="who">
```
Comment (rgaiacs) [inline _includes/dc/who.html]: This is OK. The styles include things for the workshop template.

File: _includes/lc/intro.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<p>
```
Comment (rgaiacs) [inline _includes/lc/intro.html]: This is OK. The styles include things for the workshop template.

File: _includes/lc/schedule.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/lc/schedule.html]: This is OK. The styles include things for the workshop template.

File: _includes/lc/syllabys.html [hunk 9]
```diff
@@ -0,0 +1,69 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/lc/syllabys.html]: This is OK. The styles include things for the workshop template.

File: _includes/lc/who.html [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<p id="who">
```
Comment (rgaiacs) [inline _includes/lc/who.html]: This is OK. The styles include things for the workshop template.

File: _includes/sc/intro.html [hunk 11]
```diff
@@ -0,0 +1,18 @@
+<p>
```
Comment (rgaiacs) [inline _includes/sc/intro.html]: This is OK. The styles include things for the workshop template.

File: _includes/sc/schedule.html [hunk 12]
```diff
@@ -0,0 +1,24 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/sc/schedule.html]: This is OK. The styles include things for the workshop template.

File: _includes/sc/syllabus.html [hunk 13]
```diff
@@ -0,0 +1,96 @@
+<div class="row">
```
Comment (rgaiacs) [inline _includes/sc/syllabus.html]: This is OK. The styles include things for the workshop template.

File: _includes/sc/who.html [hunk 14]
```diff
@@ -0,0 +1,8 @@
+<p id="who">
```
Comment (rgaiacs) [inline _includes/sc/who.html]: This is OK. The styles include things for the workshop template.

File: _includes/workshop_footer.html [hunk 15]
```diff
@@ -3,14 +3,22 @@
 {% endcomment %}
```
Comment (rgaiacs) [inline _includes/workshop_footer.html]: This is OK. The styles include things for the workshop template.

**Comment by rgaiacs** [comment]
> There's some library carpentry stuff as well as some commented out code from other lessons riding along, should that be there?

@gdevenyi I commented some of the changes. The Library Carpentry stuffs is because the "new" style is shared between lessons and workshop template (you can blame @gvwilson ;-P).

File: CONTRIBUTING.md (hunk 2 above)
Comment (gdevenyi) [inline CONTRIBUTING.md]: Does this mean you'll push a fix, or do I need to do this?

Comment (rgaiacs) [inline CONTRIBUTING.md]: I push a fix to this.

**Comment by gdevenyi** [comment]
Thanks!

**Pull request merged**